content/en/docs/event-sources/cloudtrail.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cloudtrail Events
+weight: 2
+---
+
+The Falco [Cloudtrail](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail#readme) plugin can read [AWS Cloudtrail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-user-guide.html) logs and emit events for each Cloudtrail log entry.
+
+Falco also distributes out-of-the-box [rules](https://github.com/falcosecurity/falco/blob/master/rules/aws_cloudtrail_rules.yaml) that can be used to identify interesting/suspicious/notable events in Cloudtrail logs, including:
+
+* Console logins that do not use multi-factor authentication
+* Disabling multi-factor authentication for users
+* Disabling encryption for S3 buckets.
+
+# Configuration
+
+The [README](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail#configuration) page has full details on how to configure the plugin. The plugin initialization and open params strings/objects can be added to `falco.yaml` under the `plugins` [configuration key](https://falco.org/docs/configuration/) key.
+
+# Ways to read Cloudtrail Logs
+
+The plugin can be configured to read log files from:
+
+* A S3 bucket
+* A SQS queue that passes along SNS notifications about new log files
+* A local filesystem path
+
+Details on the open params syntax can be found on the [open params](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail#plugin-open-params) section of the Cloudtrail README page.
+
+# Terraform Module For Cloudtrail Prerequsites
+
+In order to use the Cloutrail plugin, you must [enable](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-cloudtrail-logging-for-s3.html) Cloudtrail logging for the account(s) you want to monitor. This must be done before using the plugin.
+
+In addition, of the 3 options above, using a SQS queue provides the easiest-to-consume source of logs, as the plugin can be notified as new log files are written and automatically consume them. However, this also requires creating multiple AWS cloud resources such as SQS queues, SNS topics, etc. outside of Falco, which involve multiple manual steps.
```

Comment:
In addition, of the three options above, using an SQS queue provides the easiest-to-consume source of logs. With the SQS queue, the plugin can detect when the new log files are written and can automatically consume them. However, this also requires creating multiple AWS cloud resources, such as SQS queues, SNS topics, etc., outside of Falco, which involve multiple manual steps.

Reply:
I mostly agree with this, but want to keep the "SNS topics" in there as it emphasizes that multiple resources are required. I'll actually add some additional resources to further emphasize this.